src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -209,21 +217,35 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 				<div class="menu-item-actions description-wide submitbox">
 					<?php if ( 'custom' != $item->type && $original_title !== false ) : ?>
 						<p class="link-to-original">
-							<?php printf( __('Original: %s'), '<a href="' . esc_attr( $item->url ) . '">' . esc_html( $original_title ) . '</a>' ); ?>
+							<?php printf( __( 'Original: %s' ), '<a href="' . esc_attr( $item->url ) . '">' . esc_html( $original_title ) . '</a>' ); ?>
 						</p>
 					<?php endif; ?>
-					<a class="item-delete submitdelete deletion" id="delete-<?php echo $item_id; ?>" href="<?php
-					echo wp_nonce_url(
-						add_query_arg(
-							array(
-								'action' => 'delete-menu-item',
-								'menu-item' => $item_id,
-							),
-							admin_url( 'nav-menus.php' )
-						),
-						'delete-menu_item_' . $item_id
-					); ?>"><?php _e( 'Remove' ); ?></a> <span class="meta-sep hide-if-no-js"> | </span> <a class="item-cancel submitcancel hide-if-no-js" id="cancel-<?php echo $item_id; ?>" href="<?php echo esc_url( add_query_arg( array( 'edit-menu-item' => $item_id, 'cancel' => time() ), admin_url( 'nav-menus.php' ) ) );
-						?>#menu-item-settings-<?php echo $item_id; ?>"><?php _e('Cancel'); ?></a>
+					<a class="item-delete submitdelete deletion" id="delete-<?php echo $item_id; ?>" href="
+																					   <?php
+																						echo wp_nonce_url(
+																							add_query_arg(
+																								array(
+																									'action'    => 'delete-menu-item',
+																									'menu-item' => $item_id,
+																								),
+																								admin_url( 'nav-menus.php' )
+																							),
+																							'delete-menu_item_' . $item_id
+																						);
+																						?>
+																						"><?php _e( 'Remove' ); ?></a> <span class="meta-sep hide-if-no-js"> | </span> <a class="item-cancel submitcancel hide-if-no-js" id="cancel-<?php echo $item_id; ?>" href="
+																						<?php
+																						echo esc_url(
+																							add_query_arg(
+																								array(
+																									'edit-menu-item' => $item_id,
+																									'cancel'         => time(),
+																								),
+																								admin_url( 'nav-menus.php' )
+																							)
+																						);
+																						?>
+						#menu-item-settings-<?php echo $item_id; ?>"><?php _e( 'Cancel' ); ?></a>
```

Comment:
Is the indentation here correct? It looks weird with all that space.

Reply:
Agreed, that looks like excess indentation.

Reply:
PR updated